src/Style/ItemStyleInterface.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace PhpSchool\CliMenu\Style;
+
+interface ItemStyleInterface
```

Comment:
I think I'd rather get rid of the interfaces for each item style type. Each item renders itself so therefore can just hint on the concrete. I don't think we need interfaces for that.